public_html/wp-content/mu-plugins/blocks/assets/src/speakers/speakers-select.js

```diff
@@ -18,6 +18,10 @@ const { __ } = wp.i18n;
 import AvatarImage from '../shared/avatar';
 import VersatileSelect from '../shared/versatile-select';
 
+// lots of stuff here, can/should some of it be black-boxed into VersitileSelect, so it's DRY when we start using this component in other blocks?
+
+// this is pretty big, could it be broken reasonably down into smaller components?
+
```

Comment:
It is kind of big, but as far as I can tell, all of it is fairly specific to the Speakers block. Happy to take suggestions for stuff that could be black-boxed, though.